Tiltfile

```diff
@@ -7,22 +7,13 @@ docker_build(
 )
 
 docker_build(
-  'local:log-service',
+  'local:logservice',
   '.',
   only=['go/'],
   dockerfile='./go/Dockerfile',
   target='logservice'
 )
 
-
-docker_build(
-  'local:sysdb-migration',
-  '.',
-  only=['go/'],
-  dockerfile='./go/Dockerfile.migration',
-  target='sysdb-migration'
-)
-
```

Comment:
I just re-ordered these in the file so that it goes `service` and then `service-migration` right after it so that it reads more consistently.